server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -383,19 +383,21 @@ public ThreadPoolStats stats() {
             long rejected = -1;
             int largest = -1;
             long completed = -1;
-            if (holder.executor() instanceof ThreadPoolExecutor) {
-                ThreadPoolExecutor threadPoolExecutor = (ThreadPoolExecutor) holder.executor();
+            long waitTime = -1;
+            if (holder.executor() instanceof OpenSearchThreadPoolExecutor) {
+                OpenSearchThreadPoolExecutor threadPoolExecutor = (OpenSearchThreadPoolExecutor) holder.executor();
                 threads = threadPoolExecutor.getPoolSize();
                 queue = threadPoolExecutor.getQueue().size();
                 active = threadPoolExecutor.getActiveCount();
                 largest = threadPoolExecutor.getLargestPoolSize();
                 completed = threadPoolExecutor.getCompletedTaskCount();
+                waitTime = threadPoolExecutor.getPoolWaitTime();
```

Comment:
```
waitTime = (threadPoolExecutor instanceof OpenSearchThreadPoolExecutor) ? (OpenSearchThreadPoolExecutor)threadPoolExecutor.getPoolWaitTime() : waitTime;
```

Reply:
Hm ... @sohami why do we need another `(threadPoolExecutor instanceof OpenSearchThreadPoolExecutor)` ? we just casted it above:

```
OpenSearchThreadPoolExecutor threadPoolExecutor = (OpenSearchThreadPoolExecutor) holder.executor();
```

Reply:
I think this was supposed to be combined with the change recommended above in https://github.com/opensearch-project/OpenSearch/pull/9681#discussion_r1319235410 but since we don't need that change I don't think we need this one either.